CryptoPkg/Library/BaseCryptLib/Hmac/CryptHmacMd5.c

```diff
@@ -65,7 +65,8 @@ HmacMd5Init (
   //
   // OpenSSL HMAC-MD5 Context Initialization
   //
-  HMAC_CTX_init (HmacMd5Context);
+  memset(HmacMd5Context, 0, sizeof(HMAC_CTX));
+  HMAC_CTX_reset (HmacMd5Context);

```

Comment:
This is quite horrid, of course. In OpenSSL 1.1 they have made the ```HmacMd5Context``` an opaque structure. We *can* include internal headers and poke at it directly, but we really shouldn't.

The problem is that we have exposed the structure to our callers — or at least its size, via the ```HmacMd5GetContextSize()``` function. Can actually change the ```BaseCryptLib``` API to stop doing that? Failing that, maybe an alternative is to make it the size of a pointer. And then to do the allocation internally of the *actual* structure?